client/reducers/repos.js

```diff
@@ -83,7 +87,7 @@ export default function repos(state = {
           return Object.assign({}, state, {
             isFetching: false,
             error: false,
-            items: action.payload.items,
+            items: action.payload.items.map(repo => Object.assign(repo, {checks: initChecks(repo)})),
```

Comment:
`repos` is the reducer for the repository collection. `repo` is the reducer for a single repository. When we want to modify a single repo in the collection we should use that second reducer, like so:

```javascript
items: [
  ...state.items.slice(0, i),
  repo(state.items[i], action),
  ...state.items.slice(i + 1)
]
```